metsim/metsim.py

```diff
@@ -25,7 +25,7 @@
 now = tm.ctime(tm.time())
 user = getuser()
 
-attrs = {'pet': {'units': 'mm d-1', 'long_name': 'potential evaporation',
+attrs = {'pet': {'units': 'cm d-1', 'long_name': 'potential evaporation',
```

Comment:
That's a weird unit. Please make this `mm d-1` and do the conversion as required.